provision/acc_provision/acc_provision.py

```diff
@@ -396,6 +403,170 @@ def normalize_cidr(cidr):
     return str(n)
 
 
+def normalize_vlans(secondary_vlans):
+    normalized_vlans = []
+    if not secondary_vlans:
+        return normalized_vlans
+    for vlan in secondary_vlans:
+        if isinstance(vlan, list):
+            normalized_vlans.extend(vlan)
+        elif ',' in str(vlan):
+            values = [int(val) for val in vlan.split(',')]
+            normalized_vlans.extend(values)
```

Comment:
Let the secondary_vlan list be a string. internal to the string, lets not use quotes.. like "5-10", so result should look like: "[1,2,3,5-10]" in aci-additional-vlans in aci-containers-config configmap

Reply:
Resolved.